content/api/v18/addons.en.md

```diff
@@ -2,20 +2,20 @@
 title: 'addons'
 displayTitle: 'C++ addons'
 category: 'api'
-editPage: 'https://github.com/nodejs/node/blob/v18.8.0/doc/api/addons.md'
+editPage: 'https://github.com/nodejs/node/blob/v18.9.0/doc/api/addons.md'
```

Comment:
Aside, this could probable be guess/calculated at build time to reduce git noise

Reply:
Also missing the i18n part of the file path right now

Reply:
> Also missing the i18n part of the file path right now

WDYM? The source has no i18n. The i18n is done here.

Reply:
> Aside, this could probable be guess/calculated at build time to reduce git noise

Sure, but this PR is not the right place (IMHO) to talk about this. I'd recommend opening an issue.

Reply:
Oh, I see it's on the node repo. The URL doesn't actually take you to an editable page though, because you can't edit files on a tag path